performance/README.md

```diff
@@ -0,0 +1,55 @@
+# Setup
+To run locally first install k6. The browser module is now included in the default installation as of k6 version 0.52.0.
+`brew install k6`
+
+To run the tests head/headless you will need chromium installed - to install chromium run
+`brew install --force chromium --no-quarantine ` 
+
+# Configure the tests
+The tests are configured so the browser and api tests can be run independently or in parallel with one selected API scenario (like "load","soak", etc), to monitor the UI whilst the API is being subjected to various loads. The logic is as follows:
+* The SCENARIO env var controls the scenarios
+ 
+The available runtime scenarios are:
+ 
+ - BROWSER TEST
+ - API TESTS
+ * load
+ * soak
+ * smoke
+ * stress
+
+# Running the tests
+To run the tests locally these can be executed from the command line in a terminal. 
+From the route of the project, you can run the following commands, or navigate to the directory of the run_test.sh file and run that (you can configure which scenario to run in there too).
```

Comment:
`root` of the project

Reply:
Ah! thank you @pipwilson :D

Reply:
No idea why I spelled it that way here XD